install.sh

```diff
@@ -298,6 +298,7 @@ nvm_do_install() {
   INSTALL_DIR="$(nvm_install_dir)"
 
   SOURCE_STR="\nexport NVM_DIR=\"$INSTALL_DIR\"\n[ -s \"\$NVM_DIR/nvm.sh\" ] && \\. \"\$NVM_DIR/nvm.sh\"  # This loads nvm\n"
+  SOURCE_STR2="[ -s \"\$NVM_DIR/bash_completion\" ] && [ -n \"\$BASH_VERSION\" ] && [ -n \"\$ZSH_VERSION\" ] && \\. \"\$NVM_DIR/bash_completion\"  # This loads nvm bash_completion\n"
```

Comment:
we're only adding `SOURCE_STR2` when it's a bash or zsh file - i think this line can just assume that when it's added, it's running in bash or zsh